packages/manager/modules/pci/src/projects/project/private-registry/list/index.js

```diff
@@ -0,0 +1,59 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import listComponent from './list.component';
+
+const moduleName = 'ovhManagerPciProjectPrivateRegistryList';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+  ])
+  .config(/* @ngInject */($stateProvider) => {
+    $stateProvider.state('pci.projects.project.private-registry.list', {
+      url: '/list',
```

Comment:
You should have `pci.projects.project.private-registry` that display the private registry list (this code should be in `packages/manager/modules/pci/src/projects/project/private-registry`)